strawberryfields/backends/gaussianbackend/backend.py

```diff
@@ -102,24 +118,36 @@ def measure_homodyne(self, phi, mode, select=None, **kwargs):
         Returns:
             float: measured value
         """
-        eps = kwargs.get('eps', 0.0002)
+        if select is not None and shots != 1:
+            raise ValueError("Post selection and multiple shots is not implemented")
```

Comment:
Is there a reason not to implement this? It should return the selected value `shots` times, right?

Reply:
I think the question is again, after doing many shots, based on which of them should you collapse the rest of the register

Reply:
But the same postselection should apply to every shot, right? So in this case each shot should return the same postselected value, and collapse the state similarly?

Reply:
Deferring for now because I'd like us to think a little bit more about how to combine postselection and shots before implementing. 

Currently the backend just fixes the measurement result and updates the conditional state (i.e., no sampling done). However, with shots, a user might want to take 1000 samples, then return only those with some postselected value. This could be useful, e.g., for estimating the postselection probability